
hand_controller:
    type: position_controllers/JointTrajectoryController
    joints:
        - suc_finger_joint
        - suc_fingertip_joint
    gains:
        suc_finger_joint: {p: 100.0, i: 0.0, d: 0.0, i_clamp: 0.0}
        suc_fingertip_joint: {p: 100.0, i: 0.0, d: 0.0, i_clamp: 0.0}
    constraints:
        suc_finger_joint: {trajectory: 0.1, goal: 0.1}
        suc_fingertip_joint: {trajectory: 0.1, goal: 0.1}
    stop_trajectory_duration: 0.5
    state_publish_rate:  25
    action_monitor_rate: 10

joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50
